File: Python/Psu.py
from Komponen import Komponen

class Psu(Komponen):
    def __init__(self, watt, sertifikasi, merk, nama):
        super().__init__(merk, nama)
        self._watt = watt
        self._sertifikasi = sertifikasi

    def get_watt(self):
        return self._watt

    def set_watt(self, watt):
        self._watt = watt

    def get_sertifikasi(self):
        return self._sertifikasi

    def set_sertifikasi(self, sertifikasi):
        self._sertifikasi = sertifikasi

File: CPP/Psu.cpp
#pragma once
#include <iostream>
#include "Komponen.cpp"

using namespace std;

class Psu : public Komponen {
private:
    int watt;
    string sertifikasi;
public:
    Psu() {}

    Psu(int watt, string merk, string nama, string sertifikasi)
        : Komponen(merk, nama) {
        this->watt = watt;
        this->sertifikasi = sertifikasi;
    }

    void setWatt(int watt) { this->watt = watt; }
    void setSertifikasi(int sertifikasi) { this->sertifikasi = sertifikasi; }
    
    int getWatt() { return watt; }
    string getSertifikasi() { return sertifikasi; }

    ~Psu() {}
};
